src/ImageSharp/ColorSpaces/Conversion/Implementation/SRgbCompanding.cs

```diff
@@ -14,7 +14,7 @@ namespace SixLabors.ImageSharp.ColorSpaces.Conversion.Implementation.RgbColorSap
     /// <see href="http://www.brucelindbloom.com/index.html?Eqn_RGB_to_XYZ.html"/>
     /// <see href="http://www.brucelindbloom.com/index.html?Eqn_XYZ_to_RGB.html"/>
     /// </remarks>
-    internal class SRgbCompanding : ICompanding
+    public sealed class SRgbCompanding : ICompanding
```

Comment:
We have the same code in `Vector4Extensions`. Might make sense to have a unified & expressive common API.